microsoft/PowerToys
# [Settings] React on OS theme change fix

## Summary of the Pull Request
Register theme listener properly to react to OS theme change immediately.


## PR Checklist

- [x] **Closes:** #26791


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
 - Open Settings app
 - Ensure Theme setting is set to "Windows default"
 - Change OS Theme settings and observe that Settings app changed it's theme properly
 - Ensure that different Theme settings work as expected
